.github/workflows/main.yml

```diff
@@ -24,7 +24,9 @@ jobs:
           pip3 install pytest
```

Comment:
On a separate trivial point, this job is named `linter`

Reply:
Oh yeah so it is.  I had not noticed that there was a name XD.